### PR TITLE
fix(tutor): formatting, layout

### LINKED ISSUE #15088: Tutor: Long, oddly-spaced lines
The following two lines have a slight problem at the moment (on nightly). I am not certain that these lines are supposed to break the 80 column limit, and they have a weird amount of space in them (implying to me that two lines were accidentally merged). It's been a while since I properly went through the tutor so it's entirely possible this is intentional, but I don't recall this being a feature of any particular lesson.

https://github.com/neovim/neovim/blob/02bf251bb3f1dd55a2c6599790ae6a7772082cdd/runtime/tutor/en/vim-01-beginner.tutor#L494
https://github.com/neovim/neovim/blob/02bf251bb3f1dd55a2c6599790ae6a7772082cdd/runtime/tutor/en/vim-01-beginner.tutor#L672

### PR DESCRIPTION
**Pull request by rktjmp**
**What**

PR has two commits effecting the `:Tutor`.

One to fix #15088 and another which:

- unifies the formatting/layout, which was a bit inconsistent,
- adds a `nowrap` modeline

**Why**

See https://github.com/neovim/neovim/issues/15088 and the current tutor file was just a bit messy in spots.

Since the tutor uses a lot of conceals, which *are* included in the character count when calculating line wrapping, lines were breaking at what looked like odd spots, which gives a poor first impression and lowered readability.

I have adjusted some lines to be over 80ch in the source, but once they're rendered out with conceals, they're actually under 80, so even with `nowrap` we don't visually extend past 80.

### CODE REVIEW

**Comment by rktjmp** [comment]
@clason Sorry for the ping, can you ... ping someone who can review and close/merge this & the related issue?

**Comment by clason** [comment]
LGTM. Just for the record: this is not a file we're tracking from upstream?

**Comment by rktjmp** [comment]
From what I could tell Neovims tutor is a rewrite with more interactivity.

https://github.com/vim/vim/blob/master/runtime/tutor/tutor was pretty meaningfully different.

File: runtime/tutor/en/vim-01-beginner.tutor [hunk 1]
```diff
@@ -20,8 +20,7 @@ pressing [<Esc>](<Esc>) and then [u](u) will undo the latest change.
 This tutorial is interactive, and there are a few things you should know.
 - Type [<Enter>](<Enter>) on links [like this](holy-grail    ) to open the linked help section.
 - Or simply type [K](K) on any word to find its documentation!
-- Sometimes you will be required to modify text like
-this here
```
Comment (janlazo) [inline runtime/tutor/en/vim-01-beginner.tutor:24]: This seems to be intentional but `this here` on a separate line is awkward.
I'd expect this line to have linebreaks before and after this line and indented to be make it standout.

File: runtime/tutor/en/vim-01-beginner.tutor [hunk 2]
```diff
@@ -880,12 +864,13 @@ NOTE: If you want to ignore case for just one search command, use [\c](/\c)
...
 ** Use the on-line help system. **
 
-Vim has a comprehensive on-line help system. To get started, try one of
-these three:
-    - press the `<HELP>`{normal} key (if you have one)
-    - press the `<F1>`{normal} key (if you have one)
-    - type
-        `:help`{vim}
+Vim has a comprehensive on-line help system.
+
```
Comment (janlazo) [inline runtime/tutor/en/vim-01-beginner.tutor:868]: Extra linebreak?

File: runtime/tutor/en/vim-01-beginner.tutor [hunk 3]
```diff
@@ -924,17 +911,15 @@ default. To start using more features you have to create a "vimrc" file.
...
- 3. Press `<C-d>`{normal} and Vim will show a list of commands that start
-     with "e".
+ 3. Press `<C-d>`{normal} and Vim will show a list of commands beginning with "e".
 
  4. Press `<Tab>`{normal} and Vim will complete the command name to ":edit".
 
  5. Now add a space and the start of an existing file name: `:edit FIL`{vim}
 
- 6. Press `<Tab>`{normal}. Vim will complete the name (if it is unique).
+ 6. Press `<Tab>`{normal}. Vim will complete the name (FIL -> FILE, if it is unique).
```
Comment (janlazo) [inline runtime/tutor/en/vim-01-beginner.tutor:920]: `Vim will complete the name FIL to FILE if it is unique.`

You can make FIL,FILE standout with quotes or special characters to make them standout.

**Comment by rktjmp** [comment]
Review changes applied.

By the way, the expect system is actually broken in the tutor (broken before this PR).

Because the tutor asks you to delete lines in lesson 2.6, every expect after that point aren't checked against the correct line.

You could change the expect lines to match the "post delete" line numbers but then the initial x marks are in the wrong places.

Somewhat ironically, lesson 2.7 is undo, which could be used to undo the `dd` commands and fix the line number alignment, requires re-writing that section, or some section could instruct the user to add 3 lines somehow.

I did look at patching the plugin, but it needs some reasonable work that I didn't want to take ownership of.

(Probably needs to track applying the initial tick/cross marks to different lines than it checks the contents against, or ideally, maybe modern API's can be better used to attach both bits of data to in-buffer lines, invisible ext marks or something?)

**Comment by justinmk** [comment]
> By the way, the expect system is actually broken in the tutor (broken before this PR).

cc @fmoralesc

**Comment by fmoralesc** [comment]
@justinmk I wanted to change the whole mechanism of the tutor so that it used extmarks (and replace the use of signs with virtual text) to keep things more robust, but I haven't had the time to do so.

**Pull request merged**